iris_grib/_load_convert.py

```diff
@@ -2389,13 +2392,25 @@ def product_definition_section(section, metadata, discipline, tablesVersion,
 
     # Translate GRIB2 phenomenon to CF phenomenon.
     if tablesVersion != _CODE_TABLES_MISSING:
-        translate_phenomenon(metadata, discipline,
-                             section['parameterCategory'],
-                             section['parameterNumber'],
-                             section['typeOfFirstFixedSurface'],
-                             section['scaledValueOfFirstFixedSurface'],
-                             section['typeOfSecondFixedSurface'],
-                             probability=probability)
+        translation_kwargs = {
+            'metadata': metadata,
+            'discipline': discipline,
+            'parameterCategory': section['parameterCategory'],
+            'parameterNumber': section['parameterNumber'],
+            'probability': probability
+        }
+
+        # Won't always be able to populate the below arguments -
+        # missing from some template definitions.
+        for section_key in [
+            'typeOfFirstFixedSurface',
+            'scaledValueOfFirstFixedSurface',
+            'typeOfSecondFixedSurface'
+        ]:
```

Comment:
@trexfeathers Could you put this in a separate list i.e.
```python
        section_keys = ['typeOfFirstFixedSurface',
                        'scaledValueOfFirstFixedSurface',
                        'typeOfSecondFixedSurface']

for section_key in section_keys:
    ....
```

Reply:
👍